master/include/comm/message.hpp

```diff
@@ -12,14 +12,14 @@
 
 // PC
 #define AS_CU_EMERGENCY_SIGNAL 0x43 // TODO(andre): change or confirm code
-#define MISSION_FINISHED 0x42 // TODO(andre): change or confirm code
-#define PC_ALIVE 0x41 // TODO(andre): change or confirm code
+#define MISSION_FINISHED 0x42       // TODO(andre): change or confirm code
+#define PC_ALIVE 0x41               // TODO(andre): change or confirm code
 
 // Sensors
-#define RIGHT_WHEEL 0x11 // TODO(andre): change or confirm code
+#define RIGHT_WHEEL 0x11    // TODO(andre): change or confirm code
```

Comment:
Why not constexpr? If you are using it in the digital file

Reply:
this was me, i forgot to revert the constexpr to #define since when building it complains about variables (didn't really understand but #define fixes it)